lib/odf-report/images.rb

```diff
@@ -28,7 +28,7 @@ def update_images(file, original_zip_file)
               puts "ORIGINAL: #{new_image.base_columns.to_f}, #{new_image.base_rows.to_f}"
               puts "NEW: #{width}, #{height}"
               new_image.resize_to_fit!(width, height)
-              empty_img = ::Magick::Image.new(width, height) { self.background_color = 'rgba(255,255,255,0)' }
+              empty_img = ::Magick::Image.new(width, height) { background_color = 'rgba(255,255,255,0)' }
```

Comment:
I could not find the block syntax i nthe docs.

Can we not use the "fill" parameter as per docs?